policy/executor/internal/nodes.go

```diff
@@ -316,8 +320,13 @@ func (nodeData *ReportingQueryNodeData) score() *policy.Score {
 		}
 
 		if cur.Data.Error != nil {
-			allSkipped = false
-			foundError = true
+			var k8sNotFoundErr *k8s_common.K8sResourceNotFound
```

Comment:
we should avoid making this code dependent on k8s specific logic. Can we create an error of type asset disconnected or something like that

Reply:
What about this? https://github.com/mondoohq/cnquery/pull/380/files

Reply:
Updated the PR to include the new cnquery. Now I check for the general error.